# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-09-02 16:33+1000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: nikkurie <@nikkurie>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: ja-JP\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: selfdrive/controls/lib/events.py:151
msgid "openpilot Unavailable"
msgstr "オープンパイロットは利用できません"

#: selfdrive/controls/lib/events.py:158 selfdrive/controls/lib/events.py:165
msgid "TAKE CONTROL IMMEDIATELY"
msgstr "すぐにハンドルを持って"

#: selfdrive/controls/lib/events.py:185 selfdrive/controls/lib/events.py:335
#: selfdrive/controls/lib/events.py:361 selfdrive/controls/lib/events.py:425
#: selfdrive/controls/lib/events.py:477 selfdrive/controls/lib/events.py:529
#: selfdrive/controls/lib/events.py:539
msgid "TAKE CONTROL"
msgstr "ハンドルを持って"

#: selfdrive/controls/lib/events.py:186
#, fuzzy, python-format
msgid "Steer Unavailable Below %d %s"
msgstr "横の制御が無効になり速度が以下になります"

#: selfdrive/controls/lib/events.py:194
#, fuzzy, python-format
msgid "Calibration in Progress: %d%%"
msgstr "キャリブレーション中:"

#: selfdrive/controls/lib/events.py:195
#, fuzzy, python-format
msgid "Drive Above %d %s"
msgstr "制限速度以上の運転をしてください"

#: selfdrive/controls/lib/events.py:202
msgid "Poor GPS reception"
msgstr "GPS受信不良"

#: selfdrive/controls/lib/events.py:203
msgid "If sky is visible, contact support"
msgstr "地下・トンネルでない場合は、カスタマーサービスに連絡ください"

#: selfdrive/controls/lib/events.py:203
msgid "Check GPS antenna placement"
msgstr "GPSアンテナの位置を確認してください。"

#: selfdrive/controls/lib/events.py:208
msgid "Cruise Mode Disabled"
msgstr "クルーズモードをオフ"

#: selfdrive/controls/lib/events.py:210
msgid "Main Switch Off"
msgstr "メインスイッチをオフ"

#: selfdrive/controls/lib/events.py:220
msgid "DEBUG ALERT"
msgstr "テストメッセージを削除"

#: selfdrive/controls/lib/events.py:228
msgid "Be ready to take over at any time"
msgstr "いつでも引き継げるよう準備しておいてください。"

#: selfdrive/controls/lib/events.py:229 selfdrive/controls/lib/events.py:245
#: selfdrive/controls/lib/events.py:253 selfdrive/controls/lib/events.py:261
msgid "Always keep hands on wheel and eyes on road"
msgstr "常にハンドルに触れ、道路から目を離さない"

#: selfdrive/controls/lib/events.py:236
msgid "WARNING: White panda is deprecated"
msgstr "警告: ホワイトパンダは推奨されていません。"

#: selfdrive/controls/lib/events.py:237 selfdrive/controls/lib/events.py:277
msgid "Upgrade to comma two or black panda"
msgstr "コンマ2やブラックパンダにアップグレード"

#: selfdrive/controls/lib/events.py:244
msgid "WARNING: This branch is not tested"
msgstr "警告: このブランチはテストされていません"

#: selfdrive/controls/lib/events.py:252
msgid "Dashcam mode"
msgstr "ダッシュカムモード"

#: selfdrive/controls/lib/events.py:260
msgid "Dashcam mode for unsupported car"
msgstr "未対応車のためダッシュカムモードのみ"

#: selfdrive/controls/lib/events.py:268
msgid "Unsupported Giraffe Configuration"
msgstr "サポートされていないGiraffeの設定"

#: selfdrive/controls/lib/events.py:269
msgid "Visit comma.ai/tg"
msgstr "comma.ai/tg を参照"

#: selfdrive/controls/lib/events.py:276
msgid "White Panda Is No Longer Supported"
msgstr "ホワイトパンダはサポート終了しました"

#: selfdrive/controls/lib/events.py:280
msgid "White panda is no longer supported"
msgstr "ホワイトパンダはサポート終了しました"

#: selfdrive/controls/lib/events.py:285
msgid "Stock LKAS is turned on"
msgstr "純正LKASがオン"

#: selfdrive/controls/lib/events.py:286
msgid "Turn off stock LKAS to engage"
msgstr "純正LKASをオフにしてエンゲージ"

#: selfdrive/controls/lib/events.py:295
msgid "Community Feature Detected"
msgstr "コミュニティ開発の機能を検出"

#: selfdrive/controls/lib/events.py:296
msgid "Enable Community Features in Developer Settings"
msgstr "開発者設定でコミュニティ機能を有効にする"

#: selfdrive/controls/lib/events.py:303
msgid "Dashcam Mode"
msgstr "ダッシュカムモード"

#: selfdrive/controls/lib/events.py:304
msgid "Car Unrecognized"
msgstr "認識できない車"

#: selfdrive/controls/lib/events.py:311 selfdrive/controls/lib/events.py:319
#: selfdrive/controls/lib/events.py:327
msgid "BRAKE!"
msgstr "ブレーキ！"

#: selfdrive/controls/lib/events.py:312
msgid "Stock AEB: Risk of Collision"
msgstr "衝突の危険"

#: selfdrive/controls/lib/events.py:320
msgid "Stock FCW: Risk of Collision"
msgstr "衝突の危険"

#: selfdrive/controls/lib/events.py:328
msgid "Risk of Collision"
msgstr "衝突の危険"

#: selfdrive/controls/lib/events.py:336
msgid "Lane Departure Detected"
msgstr "車線逸脱を検知"

#: selfdrive/controls/lib/events.py:345
msgid "openpilot will not brake while gas pressed"
msgstr "アクセル中、オープンパイロットはブレーキをかけません"

#: selfdrive/controls/lib/events.py:353
msgid "Vehicle Parameter Identification Failed"
msgstr "車両パラメータの識別に失敗しました。"

#: selfdrive/controls/lib/events.py:362 selfdrive/controls/lib/events.py:530
#: selfdrive/controls/lib/events.py:533
msgid "Steering Temporarily Unavailable"
msgstr "ステアリングは一時的に利用不可"

#: selfdrive/controls/lib/events.py:369
msgid "KEEP EYES ON ROAD: Driver Distracted"
msgstr "道路から目を離さないで:注意散漫です"

#: selfdrive/controls/lib/events.py:377
msgid "KEEP EYES ON ROAD"
msgstr "道路から目を離さないで"

#: selfdrive/controls/lib/events.py:378
msgid "Driver Appears Distracted"
msgstr "ドライバーは注意散漫に見えます"

#: selfdrive/controls/lib/events.py:385 selfdrive/controls/lib/events.py:409
msgid "DISENGAGE IMMEDIATELY"
msgstr "すぐに解除してください"

#: selfdrive/controls/lib/events.py:386
msgid "Driver Was Distracted"
msgstr "ドライバーは注意力散漫"

#: selfdrive/controls/lib/events.py:393
msgid "TOUCH STEERING WHEEL: No Face Detected"
msgstr "ハンドルに触れて:顔が検出できない"

#: selfdrive/controls/lib/events.py:401
msgid "TOUCH STEERING WHEEL"
msgstr "ハンドルに触れて"

#: selfdrive/controls/lib/events.py:402
msgid "Driver Is Unresponsive"
msgstr "ドライバーが無反応"

#: selfdrive/controls/lib/events.py:410
msgid "Driver Was Unresponsive"
msgstr "ドライバーが無反応でした"

#: selfdrive/controls/lib/events.py:417
msgid "CHECK DRIVER FACE VISIBILITY"
msgstr "ドライバーの顔の視認性を確認"

#: selfdrive/controls/lib/events.py:418
msgid "Driver Monitor Model Output Uncertain"
msgstr "ドライバー監視モデルが不完全"

#: selfdrive/controls/lib/events.py:426
msgid "Resume Driving Manually"
msgstr "手動で運転を再開"

#: selfdrive/controls/lib/events.py:433
msgid "STOPPED"
msgstr "停止"

#: selfdrive/controls/lib/events.py:434
msgid "Press Resume to Move"
msgstr "Resumeを押して移動します。"

#: selfdrive/controls/lib/events.py:445
msgid "Steer Left to Start Lane Change"
msgstr "左ハンドルで車線変更を開始"

#: selfdrive/controls/lib/events.py:446 selfdrive/controls/lib/events.py:454
#: selfdrive/controls/lib/events.py:462 selfdrive/controls/lib/events.py:470
msgid "Monitor Other Vehicles"
msgstr "他の車両を監視"

#: selfdrive/controls/lib/events.py:453
msgid "Steer Right to Start Lane Change"
msgstr "右ハンドルで車線変更を開始"

#: selfdrive/controls/lib/events.py:461
msgid "Car Detected in Blindspot"
msgstr "ブラインドスポットで車両を発見"

#: selfdrive/controls/lib/events.py:469
msgid "Changing Lane"
msgstr "レーンチェンジ中"

#: selfdrive/controls/lib/events.py:478
msgid "Turn Exceeds Steering Limit"
msgstr "ステアリングリミットを超えています"

#: selfdrive/controls/lib/events.py:503
msgid "Brake Hold Active"
msgstr "サイドブレーキが作動"

#: selfdrive/controls/lib/events.py:508
msgid "Park Brake Engaged"
msgstr "サイドブレーキ作動中"

#: selfdrive/controls/lib/events.py:513
msgid "Pedal Pressed During Attempt"
msgstr "ペダル/ブレーキを検出"

#: selfdrive/controls/lib/events.py:524
msgid "Enable Adaptive Cruise"
msgstr "ACCを有効化"

#: selfdrive/controls/lib/events.py:540
msgid "Attempting Refocus: Camera Focus Invalid"
msgstr "再フォーカス中です"

#: selfdrive/controls/lib/events.py:546
msgid "Out of Storage Space"
msgstr "空き容量不足"

#: selfdrive/controls/lib/events.py:551
msgid "Speed Too Low"
msgstr "速度が遅すぎます"

#: selfdrive/controls/lib/events.py:556 selfdrive/controls/lib/events.py:560
msgid "NEOS Update Required"
msgstr "NEOSの更新が必要"

#: selfdrive/controls/lib/events.py:557
msgid "Please Wait for Update"
msgstr "更新をお待ちください"

#: selfdrive/controls/lib/events.py:565 selfdrive/controls/lib/events.py:569
msgid "No Data from Device Sensors"
msgstr "デバイスセンサからのデータがありません"

#: selfdrive/controls/lib/events.py:566 selfdrive/controls/lib/events.py:579
#: selfdrive/controls/lib/events.py:671
msgid "Reboot your Device"
msgstr "デバイスを再起動"

#: selfdrive/controls/lib/events.py:578 selfdrive/controls/lib/events.py:582
msgid "Speaker not found"
msgstr "スピーカーが見つかりません"

#: selfdrive/controls/lib/events.py:586
msgid "Distraction Level Too High"
msgstr "注意力散漫すぎます"

#: selfdrive/controls/lib/events.py:590
msgid "System Overheated"
msgstr "オーバーヒート"

#: selfdrive/controls/lib/events.py:591
msgid "System overheated"
msgstr "オーバーヒート"

#: selfdrive/controls/lib/events.py:595 selfdrive/controls/lib/events.py:596
msgid "Gear not D"
msgstr "Dではない"

#: selfdrive/controls/lib/events.py:600
msgid "Calibration Invalid: Reposition Device and Recalibrate"
msgstr "キャリブレーションが無効です。再実行してください。"

#: selfdrive/controls/lib/events.py:601
msgid "Calibration Invalid: Reposition Device & Recalibrate"
msgstr "キャリブレーションが無効です。再実行してください。"

#: selfdrive/controls/lib/events.py:605 selfdrive/controls/lib/events.py:607
msgid "Calibration in Progress"
msgstr "キャリブレーション"

#: selfdrive/controls/lib/events.py:611
msgid "Door Open"
msgstr "ドアが開いています"

#: selfdrive/controls/lib/events.py:612
msgid "Door open"
msgstr "ドアが開いています"

#: selfdrive/controls/lib/events.py:616
msgid "Seatbelt Unlatched"
msgstr "シートベルト未着用"

#: selfdrive/controls/lib/events.py:617
msgid "Seatbelt unlatched"
msgstr "シートベルト未着用"

#: selfdrive/controls/lib/events.py:621 selfdrive/controls/lib/events.py:622
msgid "ESP Off"
msgstr "ESPオフ"

#: selfdrive/controls/lib/events.py:626 selfdrive/controls/lib/events.py:627
msgid "Low Battery"
msgstr "低バッテリー"

#: selfdrive/controls/lib/events.py:631 selfdrive/controls/lib/events.py:632
msgid "Communication Issue between Processes"
msgstr "プロセス間の通信の問題"

#: selfdrive/controls/lib/events.py:637 selfdrive/controls/lib/events.py:638
msgid "Radar Communication Issue"
msgstr "レーダー通信問題"

#: selfdrive/controls/lib/events.py:643 selfdrive/controls/lib/events.py:644
#: selfdrive/controls/lib/events.py:648 selfdrive/controls/lib/events.py:649
msgid "Radar Error: Restart the Car"
msgstr "レーダーエラー:車を再起動"

#: selfdrive/controls/lib/events.py:653 selfdrive/controls/lib/events.py:654
msgid "Driving model lagging"
msgstr "制御モデルに遅延がある"

#: selfdrive/controls/lib/events.py:658 selfdrive/controls/lib/events.py:659
msgid "Vision Model Output Uncertain"
msgstr "映像が不明瞭です"

#: selfdrive/controls/lib/events.py:663 selfdrive/controls/lib/events.py:664
msgid "Device Fell Off Mount"
msgstr ""

#: selfdrive/controls/lib/events.py:668 selfdrive/controls/lib/events.py:674
msgid "Low Memory: Reboot Your Device"
msgstr "ローメモリ:デバイスを再起動"

#: selfdrive/controls/lib/events.py:670
msgid "RAM Critically Low"
msgstr "RAMが致命的に低い"

#: selfdrive/controls/lib/events.py:679 selfdrive/controls/lib/events.py:680
msgid "Controls Failed"
msgstr "制御失敗"

#: selfdrive/controls/lib/events.py:684
msgid "Controls Mismatch"
msgstr "制御不一致"

#: selfdrive/controls/lib/events.py:688 selfdrive/controls/lib/events.py:690
#: selfdrive/controls/lib/events.py:694
msgid "CAN Error: Check Connections"
msgstr "CANエラー:接続を確認"

#: selfdrive/controls/lib/events.py:698 selfdrive/controls/lib/events.py:704
msgid "LKAS Fault: Restart the Car"
msgstr "LKASの故障:車を再起動"

#: selfdrive/controls/lib/events.py:700
msgid "LKAS Fault: Restart the car to engage"
msgstr "LKASの故障:車を再起動後発進"

#: selfdrive/controls/lib/events.py:708 selfdrive/controls/lib/events.py:714
#: selfdrive/controls/lib/events.py:792
msgid "Cruise Fault: Restart the Car"
msgstr "クルーズ失敗:車を再起動"

#: selfdrive/controls/lib/events.py:710 selfdrive/controls/lib/events.py:788
msgid "Cruise Fault: Restart the car to engage"
msgstr "クルーズ失敗:車を再起動後発進"

#: selfdrive/controls/lib/events.py:718
msgid "Gas Fault: Restart the Car"
msgstr "アクセル故障:車を再起動"

#: selfdrive/controls/lib/events.py:719
msgid "Gas Error: Restart the Car"
msgstr "アクセルエラー:車を再起動"

#: selfdrive/controls/lib/events.py:723 selfdrive/controls/lib/events.py:724
msgid "Reverse Gear"
msgstr "Rに切り替え"

#: selfdrive/controls/lib/events.py:728
msgid "Cruise Is Off"
msgstr "クルーズコントロールオフ"

#: selfdrive/controls/lib/events.py:732 selfdrive/controls/lib/events.py:733
msgid "Planner Solution Error"
msgstr "Planner Solution エラー"

#: selfdrive/controls/lib/events.py:737 selfdrive/controls/lib/events.py:739
#: selfdrive/controls/lib/events.py:743
msgid "Harness Malfunction"
msgstr "ハーネスが故障"

#: selfdrive/controls/lib/events.py:740
msgid "Please Check Hardware"
msgstr "ハードウェアを確認して"

#: selfdrive/controls/lib/events.py:748 selfdrive/controls/lib/events.py:757
msgid "openpilot Canceled"
msgstr "オープンパイロットはキャンセルされました"

#: selfdrive/controls/lib/events.py:749
msgid "No close lead car"
msgstr "リードカー不在"

#: selfdrive/controls/lib/events.py:752
msgid "No Close Lead Car"
msgstr "リードカー不在"

#: selfdrive/controls/lib/events.py:758
msgid "Speed too low"
msgstr "速度が遅すぎる"

#: selfdrive/controls/lib/events.py:765 selfdrive/controls/lib/events.py:770
msgid "Speed Too High"
msgstr "速度が速すぎる"

#: selfdrive/controls/lib/events.py:766
msgid "Slow down to resume operation"
msgstr "速度を下げてオープンパイロットを再開"

#: selfdrive/controls/lib/events.py:771
msgid "Slow down to engage"
msgstr "速度を落として発進"

#: selfdrive/controls/lib/events.py:778
msgid "Please connect to Internet"
msgstr "インターネット接続を確認"

#: selfdrive/controls/lib/events.py:779
msgid "An Update Check Is Required to Engage"
msgstr "発進するには更新が必要です"

#: selfdrive/controls/lib/events.py:782
msgid "Please Connect to Internet"
msgstr "インターネット接続を確認"

#~ msgid "Left ALC will start in 3s"
#~ msgstr "左車線に移動します"

#~ msgid "Right ALC will start in 3s"
#~ msgstr "右車線に移動します"

#~ msgid "STEERING REQUIRED: Lane Keeping OFF"
#~ msgstr "操作が必要:レーンキープオフ"

#~ msgid "STEERING REQUIRED: Blinkers ON"
#~ msgstr "操作が必要:ウインカーオン"

#~ msgid "Lead Car Is Moving"
#~ msgstr "リードカーが移動しました"
